test/unit/metrics/test_metricframe_aggregates.py

```diff
@@ -1040,3 +1043,463 @@ def test_2m_1sf_2cf():
         assert ratios_overall[mfn][('m', 'aa')] == min(ratio_overall_m_a)
         assert ratios_overall[mfn][('m', 'ba')] == pytest.approx(min(ratio_overall_m_b),
                                                                  rel=1e-10, abs=1e-16)
+
+
+class Test2m1sf1cfErrorHandlingCM:
```

Comment:
Can we also have a case which has two metrics which would trigger an exception (even confusion matrix twice, once using `functools.partial` to tweak the output) and one regular metric? That would ensure that the logic handling potential exceptions works when more than one column raises.

Reply:
I am not familiar with `functools.partial`, could you please elaborate on what it does and how it can be used here? I created tests containing twice the confusion matrix metric.

Reply:
https://docs.python.org/3/library/functools.html#functools.partial would give you an idea @bram49 . You can also look for `partial` in our codebase and see how it's used in other places.

Reply:
You could do somehing like
```python
norm_conf_matrix = functools.partial(sklearn.metrics.confusion_matrix, normalize='all')
``
as the second metric function.